### PR TITLE
Update HMP Hewell capacity for Monday 15/5/23

### PR DESCRIPTION
**Pull request by AledWEvans**
## What does this pull request do?

Update HMP Hewell capacity for Monday 15/5/23

## What is the intent behind these changes?

updated capacity customer request

### CODE REVIEW

**Review by dhirajshettymoj — APPROVED** [review]
LGTM

**Review by dhirajshettymoj — APPROVED** [review]
LGTM

File: src/main/resources/session-template-data/session-data.csv [hunk 1]
```diff
@@ -23,11 +23,15 @@ HEI,Visits Main Room,SOCIAL,30,2,FALSE,09:00:00,10:00:00,2023-01-06,2023-03-20,F
 HEI,Visits Main Room,SOCIAL,30,2,FALSE,13:45:00,14:45:00,2023-01-07,2023-03-20,SATURDAY,FALSE,,,
 HEI,Visits Main Room,SOCIAL,30,2,FALSE,13:45:00,14:45:00,2023-01-08,2023-03-20,SUNDAY,FALSE,,,
 % Hewell Prison - open cap changed to 35
-HEI,Visits Main Room,SOCIAL,35,2,FALSE,13:45:00,14:45:00,2023-03-21,,MONDAY,FALSE,,,
+HEI,Visits Main Room,SOCIAL,35,2,FALSE,13:45:00,14:45:00,2023-03-21,2023-03-14,MONDAY,FALSE,,,
```
[the remark this answers is not in the document]
Comment (AledWEvans) [reply src/main/resources/session-template-data/session-data.csv]: updated

File: src/main/resources/session-template-data/session-data.csv [hunk 2]
```diff
@@ -23,11 +23,15 @@ HEI,Visits Main Room,SOCIAL,30,2,FALSE,09:00:00,10:00:00,2023-01-06,2023-03-20,F
...
 HEI,Visits Main Room,SOCIAL,35,2,FALSE,13:45:00,14:45:00,2023-03-21,,WEDNESDAY,FALSE,,,
 HEI,Visits Main Room,SOCIAL,35,2,FALSE,09:00:00,10:00:00,2023-03-21,,FRIDAY,FALSE,,,
 HEI,Visits Main Room,SOCIAL,35,2,FALSE,13:45:00,14:45:00,2023-03-21,,SATURDAY,FALSE,,,
 HEI,Visits Main Room,SOCIAL,35,2,FALSE,13:45:00,14:45:00,2023-03-21,,SUNDAY,FALSE,,,
+% Hewell Prison - 15/5/23 Open 28 tables and 4 close
+HEI,Visits Main Room,SOCIAL,28,4,FALSE,13:45:00,14:45:00,2023-03-15,2023-03-15,MONDAY,FALSE,,,
+% Hewell Prison - 16/5/23 back to normal
+HEI,Visits Main Room,SOCIAL,35,2,FALSE,13:45:00,14:45:00,2023-03-16,MONDAY,FALSE,,,
```
[the remark this answers is not in the document]
Comment (AledWEvans) [reply src/main/resources/session-template-data/session-data.csv]: updated

**Pull request merged**